src/Stripe.net/Services/SubscriptionItems/SubscriptionItemDeleteOptions.cs

```diff
@@ -0,0 +1,37 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class SubscriptionItemDeleteOptions : BaseOptions
+    {
+        /// <summary>
+        /// Delete all usage for the given subscription item. Only allowed when the current plan’s
+        /// <c>usage_type</c> is metered.
+        /// </summary>
+        [JsonProperty("clear_usage")]
+        public bool? ClearUsage { get; set; }
+
+        /// <summary>
+        /// Determines how to handle
+        /// <a href="https://stripe.com/docs/billing/subscriptions/billing-cycle#prorations">prorations</a>
+        /// when the billing cycle changes. The value defaults to
+        /// <c>create_prorations</c>, indicating that proration invoice items should
+        /// be created. Prorations can be disabled by setting the value to
+        /// <c>none</c>.
+        /// </summary>
+        [JsonProperty("proration_behavior")]
+        public string ProrationBehavior { get; set; }
```

Comment:
Not yet supported but will be by the time we merge, I don't want to add `prorate` since it will be deprecated at the same time.